src/time.cr

```diff
@@ -1216,11 +1220,12 @@ struct Time
     @seconds + offset
   end
 
-  private def year_month_day_day_year
-    m = 1
-
-    days = DAYS_MONTH
-    totaldays = offset_seconds / SECONDS_PER_DAY
+  # Returns the calendrical representation of this instance's date.
+  #
+  # The return value is a tuple consisting of year (`1..9999`), month (`1..12`),
+  # day (`1..31`) and ordinal day of the year (`1..366`).
+  protected def year_month_day_day_year : {Int32, Int32, Int32, Int32}
+    totaldays = (offset_seconds / SECONDS_PER_DAY).to_i
```

Comment:
Maybe name it `total_days`?

Reply:
I had considered that, but wasn't sure about it. Will change it :+1: